pkg/validation/release.go

```diff
@@ -57,14 +57,17 @@ func validateReleases(fieldRoot string, releases map[string]api.UnresolvedReleas
 	return validationErrors
 }
 
-func validateIntegration(fieldRoot string, integration api.Integration) []error {
+func validateIntegration(fieldRoot, name string, integration api.Integration) []error {
 	var validationErrors []error
 	if integration.Name == "" {
 		validationErrors = append(validationErrors, fmt.Errorf("%s.name: must be set", fieldRoot))
 	}
 	if integration.Namespace == "" {
 		validationErrors = append(validationErrors, fmt.Errorf("%s.namespace: must be set", fieldRoot))
 	}
+	if integration.IncludeBuiltImages && name != api.LatestReleaseName {
+		validationErrors = append(validationErrors, fmt.Errorf("%s: only the `latest` release can set `include_built_images`", fieldRoot))
```

Comment:
We would be able to relax this in the future if we stop using this shenanigans around `stable(-foo)?` streams and do something rigorous.